src/System.Private.ServiceModel/tests/Common/Infrastructure/src/BridgeClient.cs

```diff
@@ -10,7 +10,7 @@
 
 namespace Infrastructure.Common
 {
-    public static class BridgeClient 
+    internal static class BridgeClient 
```

Comment:
Reviewers: I made these internal as the 1st step in deprecating/renaming them.  We had tests that took direct dependencies on these types, and 'internal' caught all those.